pkgs/top-level/all-packages.nix

```diff
@@ -33684,6 +33684,14 @@ with pkgs;
 
   rofi-rbw = python3Packages.callPackage ../applications/misc/rofi-rbw { };
```

Comment:
Even if "redundant", please set the `enableX11` and `enableWayland` here.

Reply:
done